src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqConnectionSettings.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Net;
+using RabbitMQ.Client;
+
+namespace CoreWCF.Channels
+{
+    public class RabbitMqConnectionSettings
+    {
```

Comment:
I couldn't find this type exposed publicly anywhere. Does it need to be public?

Reply:
It remained public for unit tests and under the assumption that it would be used in anything client-related